src/deploy.ts

```diff
@@ -384,18 +524,40 @@ class Deployer {
       }
     }
 
+    let {continuousDeployment} = deployConfig;
+    if (continuousDeployment === null) {
+      const enable = await this.effects.clack.confirm({
+        message: wrapAnsi(
+          `Do you want to enable continuous deployment? ${faint(
+            "This builds in the cloud and redeploys whenever you push to this repository."
```

Comment:
Dunstan says this could note upfront that continuous deployment requires a GitHub repository

Reply:
done